apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.UnitTests/SessionPoolTests.cs

```diff
@@ -72,67 +85,165 @@ private Mock<SpannerClient> CreateMockClient(DatabaseName dbName = null)
                 .ReturnsAsync(
                     () =>
                     {
-                        var tx = new Transaction { Id = ByteString.CopyFromUtf8(Guid.NewGuid().ToString())};
+                        var tx = new Transaction {Id = ByteString.CopyFromUtf8(Guid.NewGuid().ToString())};
                         _transactions.Push(tx);
                         return tx;
                     });
 
-            mockClient.Setup(x => x.CommitAsync(It.IsAny<SessionName>(), It.IsAny<ByteString>(),
-                It.IsAny<IEnumerable<Mutation>>(), It.IsAny<CallSettings>()))
+            mockClient.Setup(
+                    x => x.CommitAsync(
+                        It.IsAny<SessionName>(), It.IsAny<ByteString>(),
+                        It.IsAny<IEnumerable<Mutation>>(), It.IsAny<CallSettings>()))
                 .ReturnsAsync(() => new CommitResponse());
 
             return mockClient;
         }
 
+        private static T[] DuplicateTaskAsync<T>(Func<T> factory, int count) =>
+            Enumerable.Range(0, count).Select(_ => factory()).ToArray();
+
+        /// <summary>
+        /// Creates <paramref name="parallelCount"/> write sessions simultaneously over
+        /// <paramref name="iterations"/> number of iterations.
+        /// </summary>
+        private async Task CreateReleaseWriteSessions(
+            SessionPool pool,
+            SpannerClient client,
+            int iterations,
+            int parallelCount)
+        {
+            await Task.Yield();
```

Comment:
Add a comment to explain this and the yield below. Someone could come along in the future and remove it, thinking it is unnecessary.